### DISCUSSION TITLE
Swag give aways

### DISCUSSION THREAD

**Discussion by eddiejaoude** [eddiejaoude/stargate#35] [Q&A] [answered]
- [x] @srinath1412001 https://github.com/eddiejaoude/stargate/issues/19
- [x] @KhushiSharma22 
- [x] @jai-dewani 
- [x] @muthuannamalai12 https://github.com/eddiejaoude/stargate/pull/28

**Comment by eddiejaoude** [comment, marked as answer]
Posted 🎉 should be with you all soon 💥

**Reply by rsrinath14** [reply]
Thanks a lot Eddie!
